tests/test_main_check_connection.py

```
@@ -0,0 +1,620 @@
+    _ImmediateEvent,
+    _OnePassEvent,
+    _reset_all_mmrelay_globals,
+    _TaskSpy,
+)
+from tests.constants import TEST_MATRIX_HOMESERVER, TEST_ROOM_ID_1
+from tests.helpers import (
+    inline_to_thread,
+    make_patched_get_running_loop,
+)
+
+if sys.version_info < (3, 10):
```

Ruff (0.15.12)

[warning] Version block is outdated for minimum Python version

Remove outdated version block

(UP036)